src/style.css

```diff
@@ -1 +1,23 @@
 /* add styles here */
+
+@import './blocks/big-search.css';
+@import './blocks/small-search.css';
+
+* {
+  margin: 0;
+  padding: 0;
+  font-family: Avenir, system-ui;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf), url(./fonts/Avenir-Heavy.ttf),
+    url(./fonts/Avenir.ttc);
+  font-weight: 300;
```

Comment:
The 'font-weight' property is set to 300, but it's unclear if this matches the actual weights available in the 'Avenir-Book.ttf', 'Avenir-Heavy.ttf', and 'Avenir.ttc' files. Verify that the specified weight is supported by these font files.